modules/ossm-security-mtls.adoc

```diff
@@ -44,6 +58,8 @@ spec:
     mode: STRICT
 ----
 
+If you are setting PeerAuthentication to STRICT and you are not using automatic mTLS, a `DestinationRule` resource is also required for your services.
```

Comment:
Would it make sense for this to be a note/warning?

Reply:
Sure, done.